README.md

```diff
@@ -1,69 +1,149 @@
 
-[![NPM][1]][2]
+[![release level](https://img.shields.io/badge/release%20level-beta-yellow.svg?style=flat)](https://cloud.google.com/terms/launch-stages)
+[![npm version](https://img.shields.io/npm/v/@google-cloud/connect-firestore.svg)](https://www.npmjs.org/package/@google-cloud/connect-firestore)
+[![codecov](https://img.shields.io/codecov/c/github/googleapis/nodejs-firestore-session/master.svg?style=flat)](https://codecov.io/gh/googleapis/nodejs-firestore-session)
 
-[1]: https://img.shields.io/npm/v/@google-cloud/connect-firestore.svg?style=flat
-[2]: https://www.npmjs.org/package/@google-cloud/connect-firestore
 
-**@google-cloud/connect-firestore** is a [Google Cloud Firestore][firestore]
-session store backed by [@google-cloud/firestore][firestore_lib].
 
-**Note:** Cloud Firestore is a persistent, distributed, transactional database.
+
+> An [express](http://expressjs.com/) session store backed by [Google Cloud Firestore][product-docs].
+
+**@google-cloud/connect-firestore** is an [express](http://expressjs.com/)
+session store backed by [Google Cloud Firestore][product-docs].
+
+**Note**: Cloud Firestore is a persistent, distributed, transactional database.
 Often, it's more appropriate to choose a different storage solution for sessions
 such as Memcache or Redis as their designs offer much faster operation in this
 use case.
 
-## Installation
 
-    npm install @google-cloud/connect-firestore
+* [Google Cloud Firestore Session Node.js Client API Reference][client-docs]
+* [Google Cloud Firestore Session Documentation][product-docs]
+* [github.com/googleapis/nodejs-firestore-session](https://github.com/googleapis/nodejs-firestore-session)
+
+Read more about the client libraries for Cloud APIs, including the older
+Google APIs Client Libraries, in [Client Libraries Explained][explained].
+
+[explained]: https://cloud.google.com/apis/docs/client-libraries-explained
+
+**Table of contents:**
+
+
+* [Quickstart](#quickstart)
+  * [Before you begin](#before-you-begin)
+  * [Installing the client library](#installing-the-client-library)
+  * [Using the client library](#using-the-client-library)
+* [Samples](#samples)
+* [Versioning](#versioning)
+* [Contributing](#contributing)
+* [License](#license)
 
-## Configuration
+## Quickstart
 
-You must have a Google Cloud project and credentials.
+### Before you begin
 
-See [gcloud node's documentation][auth] on setting up authentication.
+1.  [Select or create a Cloud Platform project][projects].
+1.  [Enable the Google Cloud Firestore Session API][enable_api].
+1.  [Set up authentication with a service account][auth] so you can access the
+    API from your local workstation.
 
-## Usage Example
+### Installing the client library
+
+```bash
+npm install @google-cloud/connect-firestore
+```
+
+
+### Using the client library
 
 ```javascript
-const Firestore = require('@google-cloud/firestore');
+const {Firestore} = require('@google-cloud/firestore');
```

Comment:
When did Firestore start to return an object with the `Firestore` field? I used it last night and it's `const Firestore = required('@google-cloud/firestore');

Reply:
Fun fact...  it supports both!